### DISCUSSION TITLE
Sovol SV07 profile for Orca?

### DISCUSSION THREAD

**Discussion by psmithrpm** [OrcaSlicer/OrcaSlicer#1997] [Q&A]
Hi folks, I'm completely a newbee and just getting into 3D printing. It looks like Orca Slicer is the way to go, and I'm wondering if anyone has already created a Sovol SV07 profile for it?  Thanks for your help! --Paul

**Comment by justBAKES** [comment]
there was a profile posted by another user in r/sovol  just today, but honestly i have no idea where to place the .json files making the printer pop up when adding a new machine to orca. i already swapped over the profile from the reskinned cura to 5.4.... but your 1000% correct- orca is the truth man. the tool integration natively is just next level. id really like to see our printer added and be able to wirelessly send to it the same way i do to my p1p

**Comment by psmithrpm** [comment]
Thank you for letting me know about today's contribution. Ya, we really
need an SV07 profile added to Orca. I'm really new to this, but I did some
nosing around in order to try to comprehend using ".json" text. It seems to
be an edit to the "config.json" that's found in the Import Menu. But beyond
that, I'm still digging. The text supplied by the user would be edited in
somehow, noting that the first chunk is the top and one or more of the
following chunks would apparently conclude the import. But that's a big
guess at this point. If I can figure this out, I'll drop another note.
Thanks again for pointing me at the .json text!

On Thu, Sep 7, 2023 at 9:18 PM justBAKES ***@***.***> wrote:

> there was a profile posted by another user in r/sovol just today, but
> honestly i have no idea where to place the .json files making the printer
> pop up when adding a new machine to orca. i already swapped over the
> profile from the reskinned cura to 5.4.... but your 1000% correct- orca is
> the truth man. the tool integration natively is just next level. id really
> like to see our printer added and be able to wirelessly send to it the same
> way i do to my p1p
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SoftFever/OrcaSlicer/discussions/1997#discussioncomment-6943005>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUXIURSRJPH23H2YSBTVE7LXZJW5TANCNFSM6AAAAAA4JVKEI4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*****************************************

*Paul H. Smith*
*Senior Editor*
***@***.***
www.AdventureMotorcycle.com
*Adventure Motorcycle Magazine*
ADVMoto  YouTube <https://www.youtube.com/c/ADVMoto>  | Instagram
<https://www.instagram.com/advmotomag/> | Facebook
<https://www.facebook.com/ADVMotoMag/>
Submission Guidelines
<https://adventuremotorcycle.com/submission-guidelines>
West Coast Office
637 N Benton Way
Los Angeles, CA 90026
Tel: In North America: (818) 679-0332

**Reply by justBAKES** [reply]
we got added bro! update to the new build its got the 7 & 7+
